tests/test_pipelines.py

```diff
@@ -31,7 +31,7 @@
 TEXT2TEXT_FINETUNED_MODELS = ["patrickvonplaten/t5-tiny-random"]
 TF_TEXT2TEXT_FINETUNED_MODELS = ["patrickvonplaten/t5-tiny-random"]
 
-DIALOGUE_FINETUNED_MODELS = ["microsoft/DialoGPT-medium"]
+DIALOGUE_FINETUNED_MODELS = ["microsoft/DialoGPT-medium"]  # @slow
```

Comment:
Why the comment here? Because it's a large model?

Reply:
I copied what was done above for `LARGE_FILL_MASK_FINETUNED_MODELS`

Reply:
(and yes it's a large model)